Formula/hapi-fhir-cli.rb

```diff
@@ -0,0 +1,154 @@
+class HapiFhirCli < Formula
+  desc "Command-line interface for the HAPI FHIR library"
```

Comment:
Not very descriptive

Reply:
According to `brew audit`, the limit is 80 characters... any suggestions? I couldn't fit anything more descriptive.